### ISSUE TITLE
Deactivating single specs or single tests via sorry-cypress Dashboard

### ISSUE THREAD

**Issue by mirobo** [sorry-cypress/sorry-cypress#761]
- [x] I have searched issues for similar feature requests

## What is this feature about (1 sentence)
I would like to be able to "deactivate" whole spec files or single tests within a spec file via the sorry-cypress Dashboard.

## Why is it needed? What is the value? For whom do we build it?
When dealing with flaky tests (even when using test retries) it is ofter cumbersome to create a branch, exclude a test, create a ticket (depending on the company's workflow) and reference the ticket in the test code. Sometimes flaky tests can evolve depending on the infrastructure, depending of the time when tests are executed (heavy load on executing agents). There are a LOT of factors playing into this. In reality we ofter delay the fix of flaky tests by deactivating them so it doesn't block other builds. We try to solve the flaky tests ASAP, but sometimes other issues have priority over this.

Also sorry-cypress could give an overview over all deactivated tests and they could be activated/deactivated on the fly for all builds (it wouldn't required a branch + merge to a release branch)

With cypress-cloud this would be quite easy to solve, because in the sorry-cypress Dashboard you could manage the excluded tests:

![image](https://user-images.githubusercontent.com/78788210/227250961-ce6f2a2a-848f-4e89-ae5d-a93dffceb147.png)

Ideally the spec files would be manipulated before the tests start, but I guess it's easier to run everything and then just tweak the result and report it into a special section of the run ("ignored results").

Maybe some code was changed which fixed some flaky tests by accident. Then they could appear in a different section "involuntarily fixed tests".

Basically: move flaky test management out of the code into sorry-cypress.

I think this could be a very useful feature (not necessarily linked to Cypress but for sorry-cypress in general) to speed up the workflow of dealing with flaky tests (or in this case delaying to deal with them ;-)).

## Other

I'm eager to know what you think of this :)

**Comment by agoldis** [comment]
@mirobo this is a great idea but honestly I can't see how can we get to it with the resources we have
